### ISSUE TITLE
Support for more well-known protobuf types

### ISSUE THREAD

**Issue by papa-whisky** [riok/Kreya#36]
I'm working with a service where the request message includes a field with `Any` type. I've noticed that when making a request I can use some of the [well-known types](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf) for this field, but not others. E.g.;

```json
"any_field": {
  "@type": "type.googleapis.com/google.protobuf.StringValue",
  "value": "fooBar"
}
```

works fine, but:

```json
"any_field": {
  "@type": "type.googleapis.com/google.protobuf.Struct",
  "value": {"foo": "bar"}
}
```

results in an error (pasted message below). I've noticed that `StringValue` and `BoolValue` work, `Struct` and `ListValue` do not. Haven't tested everything though. `grpcurl` seems to handle anything I throw at it, while BloomRPC handles nothing and I have to manually encode the value. Anyway, would be great if Kreya could handle `Struct` at least!

As an aside, I also noticed most tools allow specifying the type _without_ the `type.googleapis.com/` prefix, but Kreya doesn't. Small thing but might save some hassle if we could use the short form.

Error message:

```
{
  "kreyaVersion": "1.4.0",
  "platform": "MacIntel",
  "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko)"
}

System.InvalidOperationException: Type registry has no descriptor for type name 'google.protobuf.Struct'
   at Google.Protobuf.JsonParser.MergeAny(IMessage message, JsonTokenizer tokenizer)
   at Google.Protobuf.JsonParser.Merge(IMessage message, JsonTokenizer tokenizer)
   at Google.Protobuf.JsonParser.ParseSingleValue(FieldDescriptor field, JsonTokenizer tokenizer)
   at Google.Protobuf.JsonParser.MergeField(IMessage message, FieldDescriptor field, JsonTokenizer tokenizer)
   at Google.Protobuf.JsonParser.Merge(IMessage message, JsonTokenizer tokenizer)
   at Google.Protobuf.JsonParser.Merge(IMessage message, TextReader jsonReader)
   at Google.Protobuf.JsonParser.Parse(TextReader jsonReader, MessageDescriptor descriptor)
   at Kreya.Grpc.Core.Invoker.GrpcRequestConverter.Convert(IOperationWithDetail`2 operation, GrpcRequest request)
   at Kreya.Grpc.Core.Invoker.AbstractGrpcInvoker.SendRequest(GrpcRequest request)
   at Kreya.Core.Invoker.OperationInvoker.SendAllRequestsAndComplete()
   at SpiderEye.Bridge.ApiMethod.InvokeAsync(Object instance, Object parameter)
   at SpiderEye.Bridge.WebviewBridge.InvokeWithDependencyInjection(IDependencyInjectionApiMethod apiMethod, Object parameters)
   at SpiderEye.Bridge.WebviewBridge.InvokeWithDependencyInjection(IDependencyInjectionApiMethod apiMethod, Object parameters)
   at SpiderEye.Bridge.WebviewBridge.ResolveCall(String id, String parameters)
```

edit: Thanks for your work on Kreya, btw! 🙏

**Comment by CommonGuy** [comment]
Thanks for using Kreya and the detailed issue description :) 

Kreya currently only knows about types that are explicitly imported. Adding `import "google/protobuf/struct.proto";` to one of your proto files should work as a workaround. You could also add a new, separate protobuf file which only imports the well known type, so that you don't have to fiddle with your "productive" proto files.

I can see that this is not an optimal solution. Adding all the well known types to the type registry by default seems a good solution to me, then the workaround wouldn't be necessary anymore.

**Comment by papa-whisky** [comment]
Ah that makes sense, I am indeed importing `StringValue` and `BoolValue` via my proto files. So, I guess this will only be an issue when working with `Any` fields, I can't imagine another scenario where the type wouldn't explicitly be imported 🤔.

Thanks for the reply, the workaround will help me for now 👍

**Comment by CommonGuy** [comment]
With the newly released 1.6 version, you shouldn't need to import the well known types anymore if you want to use them with `Any`.

**Comment by oheyzhiwei** [comment]
Just encountered this when dealing with FieldMask https://developers.google.com/protocol-buffers/docs/reference/csharp/class/google/protobuf/well-known-types/field-mask
